scripts/migrate-users.py

```diff
@@ -0,0 +1,237 @@
+    cluser.first_name,
+    cluser.last_name,
+    cluser.email,
+    cluser.date_joined,
+    cluser.organization_or_affiliation AS affiliation,
+    cluser.is_superuser,
+    email.verified AS is_verified
+    FROM authenz_cluser AS cluser
+    LEFT OUTER JOIN account_emailaddress AS email
+    ON cluser.id = email.user_id
+    WHERE cluser.id != -1 AND cluser.is_active = 1""")
+django_users = list(django_cursor.fetchall())
+
+###############################################################
+# Preprocess user data
+###############################################################
+
+# Generate new user ids using uuid
+for user in django_users:
+    user['old_user_id'] = str(user.pop('id'))
+    user['new_user_id'] = uuid.uuid4().hex
+
+###############################################################
+# Get existing users in bundles db
+###############################################################
+
+# Get set of user ids in bundles db
+with model.engine.begin() as connection:
+    bundle_users = connection.execute(select([cl_user.c.user_id])).fetchall()
+
+# Find intersection of user ids
+django_user_ids = set(user['old_user_id'] for user in django_users)
+bundle_user_ids = set(user['user_id'] for user in bundle_users)
+to_update = django_user_ids & bundle_user_ids
+to_insert = django_user_ids - bundle_user_ids
+
+print "Users to update:", ', '.join(list(to_update))
+print "Users to insert:", ', '.join(list(to_insert))
+
+to_update = [user for user in django_users
+             if (user['old_user_id'] in to_update)]
+to_insert = [user for user in django_users
+             if (user['old_user_id'] in to_insert)]
+
+###############################################################
+# Update existing users in bundles db
+###############################################################
+
+# Turn on query logging
+model.engine.echo = True
+
+if to_update:
+    print "Updating existing users in bundle service database..."
+
+    try:
+        with model.engine.begin() as connection:
+            update_query = cl_user.update().\
+                where(cl_user.c.user_id == bindparam('old_user_id')).\
+                values(is_active=1, user_id=bindparam('new_user_id'))
+            connection.execute(update_query, to_update)
+
+            if dry_run:
+                raise DryRunAbort
+    except DryRunAbort:
+        pass
+
+###############################################################
+# Insert remaining users into bundles db
+###############################################################
+
+if to_insert:
+    print "Inserting new users into bundle service database..."
+
+    default_user_info = manager.default_user_info()
+
+    # Throw away old user ids
+    for user in to_insert:
+        del user['old_user_id']
+
+    try:
+        with model.engine.begin() as connection:
+            insert_query = cl_user.insert().\
+                values(user_id=bindparam('new_user_id'),
+                       time_quota=default_user_info['time_quota'],
```

Comment:
The other fields are automatically "bound" by sqlalchemy. (It inspects the keys in the first dict passed to `execute`. And it asserts that all the following dicts have exactly the same keys.)